packages/sources/css/src/components/badge/src/index.css

```diff
@@ -5,8 +5,11 @@
 .vtmn-badge {
   font-family: var(--vtmn-typo_font-family);
   font-size: var(--vtmn-typo_text-3-font-size);
+  font-weight: 500;
```

Comment:
Sorry it's impossible because we require only 400 and 700 in all our README. We need to convince designers that it's a bad practice to have a weight of 500 just for one little component as the badge.
I vote for 400 :)

Reply:
I went back for the token regular font-weight